### PR TITLE
Master website perfect ui sri

### PR DESCRIPTION
**Pull request by stefanorigano**
Website UI refinement

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web_editor/static/src/scss/web_editor.variables.scss [hunk 1]
```diff
@@ -144,11 +144,11 @@ $o-we-sidebar-content-field-dropdown-item-active-color: $o-we-dropdown-item-acti
...
-$o-we-sidebar-content-field-colorpicker-dropdown-bg: $o-we-bg-light !important;
-$o-we-sidebar-content-field-colorpicker-dropdown-color: $o-we-fg-light !important;
-$o-we-sidebar-content-field-colorpicker-dropdown-active-color: $o-we-fg-lighter !important;
-$o-we-sidebar-content-field-colorpicker-cc-width: 208px !important;
-$o-we-sidebar-content-field-colorpicker-cc-height: 26px !important;
+$o-we-sidebar-content-field-colorpicker-dropdown-bg: $o-we-bg-light !default;
+$o-we-sidebar-content-field-colorpicker-dropdown-color: $o-we-fg-light !default;
+$o-we-sidebar-content-field-colorpicker-dropdown-active-color: $o-we-fg-lighter !default;
+$o-we-sidebar-content-field-colorpicker-cc-width: 208px !default;
+$o-we-sidebar-content-field-colorpicker-cc-height: 26px !default;
```
Comment (qsm-odoo) [inline addons/web_editor/static/src/scss/web_editor.variables.scss:162]: Oups :worried:

**Comment by qsm-odoo** [comment]
Already merging half of the reviewed PR with https://github.com/odoo/odoo/pull/55645 :+1:

**Comment by qsm-odoo** [comment]
I'll rebase this one once the mentioned one is merged

**Comment by stefanorigano** [comment]
Perfect, thanks 🙏

Il ven 7 ago 2020, 19:35 Quentin Smetz <notifications@github.com> ha
scritto:

**Comment by qsm-odoo** [comment]
Push-forced the branch with the rebased version. I'll check the rest tomorrow if you confirm to me everything here (except the POC) is to be merged.

**Comment by stefanorigano** [comment]
Yes, all confirmed (except POC of course)

Il dom 9 ago 2020, 17:19 Quentin Smetz <notifications@github.com> ha
scritto:

**Comment by qsm-odoo** [comment]
Merged a lot of this PR work at https://github.com/odoo/odoo/pull/55959

@stefanorigano I push-forced the branch here, the 2 first commits are to be ignored. I made them to try to be able to show you what I changed in the commits I merged... but I kinda failed :') You can have an idea with that diff though: https://github.com/odoo/odoo/pull/55573/commits/3b5ff709d909cc4d284fc82df22fe594e127ab7b Design-wise, I think I only changed three things (if I am not mistaken): the way the dropdown arrows work (so that they work for font family picker), some styles that were not working on firefox (like the colorpicker inputs) and the 1px vertical padding you added with https://github.com/odoo/odoo/pull/55573/commits/3b5ff709d909cc4d284fc82df22fe594e127ab7b#diff-6aba36a2ef6bfd4e64e4f05ef7815a97L161 (which actually breaks the layout). On top of that, there are some remaining commits I need to consider later.

**Comment by stefanorigano** [comment]
@qsm-odoo , thanks.... for the explicative commits too :) 

I noticed issues with buttons when placed into the level's header (check attach). Also a `position: relative` in the level's header is now missing, causing the "hover bar" to fill the entire the vertical space.
For the rest seems all perfect, thanks again.

I pushed bc25f090bc39df7c27519d22659e36267fac8cb8 on top to handle the "stand-up" on the new snippet search bar.

image
image
image

**Comment by qsm-odoo** [comment]
Probably too old to consider now but if needed, can be moved in new PR :)

**Pull request closed without merging**